test/Makefile

```diff
@@ -382,6 +383,15 @@ uninstall_spiderpool:
 	@echo -e "\033[35m [helm uninstall spiderpool] \033[0m"
 	helm uninstall $(RELEASE_NAME) --wait --debug -n $(RELEASE_NAMESPACE) \
 	     --kubeconfig $(E2E_KUBECONFIG)  || { KIND_CLUSTER_NAME=$(E2E_CLUSTER_NAME) ./scripts/debugEnv.sh $(E2E_KUBECONFIG) "detail"   ; exit 1 ; } ; \
+	NODE_LIST=` kind get nodes --name $(E2E_CLUSTER_NAME) `; \
```

Comment:
I add it in  https://github.com/spidernet-io/spiderpool/pull/3716/files#diff-740cb5a1689091cb894445e46683c255e39689ccba1c6c63d8a8841c8df8817dR411

Reply:
ok,  I will rebase it after https://github.com/spidernet-io/spiderpool/pull/3716 merged.

Reply:
I mean these codes could be removed

Reply:
done